app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java

```diff
@@ -429,6 +429,17 @@ public void onCreate(Bundle savedInstanceState) {
 
     ActionBarDrawerToggle drawerToggle = new KiwixActionBarDrawerToggle(this, drawerLayout, toolbar);
 
+    Boolean isFirstRun = getSharedPreferences("PREFERENCE", MODE_PRIVATE)
```

Comment:
We have a new shared preference system now.

Reply:
actually my bad about this. It isn't in just yet. Keeping track of 25 pull requests is getting a bit out of hand :P